src/transpile.rs

```diff
@@ -684,7 +683,8 @@ impl ParamContext {
         };
 
         let val_datum = json_to_text_datum(value)?;
-        self.params.push((type_oid, val_datum));
+        let datum_with_oid = unsafe { DatumWithOid::new(val_datum, type_oid.value()) };
+        self.params.push(datum_with_oid);
         Ok(format!("(${}::{})", self.params.len(), type_name))
     }
 }
```

Comment:
All changes in this file are due to changes in signature of method in latest pgrx version.